macros/SpecName.ejs

```diff
@@ -1138,4 +1142,4 @@ if ($2) {
 }
 
 
-%><%- result %>
+%><%- result %>

```

Comment:
It's a byproduct of adding lines higher up. The initial - and + aren't part of the file. They're git's identifiers for what's changed.